core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/description.properties

```diff
@@ -0,0 +1 @@
+blurb = Warns about disabled <a href="https://www.jenkins.io/redirect/csrf-protection" rel="noopener noreferrer" target="_blank">CSRF protection</a>.
```

Comment:
Is it working without the use of `j:out`? (not manally tested)

Reply:
See screenshot.

At least in Jelly views, parameters to localized expressions are escaped, but content is not.